index.html

```diff
@@ -1928,7 +1928,7 @@ <h3>Defining 'string'</h3>
 </div>
 
 <div class="req" id="char_string_dom">
-	<p class="advisement">If your specification pertains to the [[DOM]], [[HTML]], or uses [[WebIDL]], or defines strings as opaque values that are not processed, use {{DOMString}} as a definition of string.</p>
+	<p class="advisement">Use a string definition consistent with {{DOMString}} if your specification does not process the internal value of strings and is not required to check for unpaired surrogate code points, or if your specification pertains to the [[DOM]], defines a JavaScript API or data format, or defines strings as opaque values that are not processed.</p>
 	<details class="links"><summary>explantations &amp; examples</summary>
```

Comment:
i could be wrong, but **explantations** probably needs to be changed to **explanations**

Reply:
otherwise, i think this is probably ok, thanks

Reply:
🤦

Good catch. Sadly, I *copied* that text from elsewhere in the previous doc to avoid making typos. Or is an _explantation_ an explanation from the plantation?